mart/utils/utils.py

```diff
@@ -263,12 +263,12 @@ def get_resume_checkpoint(config: DictConfig) -> Tuple[DictConfig, str]:
 
         resume_checkpoint = resume_checkpoint[0]
         log.info(f"Resuming from {resume_checkpoint}")
+        # Save the ckpt_path in cfg for fit() and test().
```

Comment:
This is begging for a test. It's not clear to me why `get_resume_checkpoint` has to change?

Reply:
We need the updated check point path from `get_resume_checkpoint()` later in `fit()` or `test()`. By writing to `cfg.ckpt_path`, we avoid changing `lightning` and `task_wrapper`.

Reply:
Okay I see.